app/models/invite_user_form.rb

```diff
@@ -1,16 +1,16 @@
 class InviteUserForm
   include ActiveModel::Model
 
-  attr_reader :email, :given_name, :family_name, :roles
+  attr_reader :email, :first_name, :last_name, :roles
 
-  validates_presence_of :email, :given_name, :family_name, :roles
+  validates_presence_of :email, :roles, :first_name, :last_name
```

Comment:
can we keep the order of what's on the screen? just so when the error messages appear they're in the same order as the fields

Reply:
thanks, have amended